roles/nova-common/defaults/main.yml

```diff
@@ -23,7 +27,7 @@ nova:
   python_libvirt_version: 1.2.2-0ubuntu2~cloud0
   qemu_kvm_version: 2.0.0+dfsg-2ubuntu1.16~cloud0
   librdb1_version: 0.80.9-0ubuntu0.14.04.2~cloud0
-  glance_endpoint: http://{{ endpoints.glance }}:9292
+  glance_endpoint: "{{ endpoints.glance.url.internal }}"
```

Comment:
nope, the keystone endpoint for glance would look like (this is define in endpoints role), showing example output from deployed cluster..
Public:        https://conv-dev.open-test.ibmcloud.com:9292
Internal: http://10.112.242.186:9393 ---> is what nova.conf is pointing to .... urcarp ip
Admin: https://conv-dev.open-test.ibmcloud.com:9292